### ISSUE TITLE
Parameter validation error

### ISSUE THREAD

**Issue by jdebacker** [PSLmodels/OG-USA#551]
As I look into Issue #550, I rebuilt my env and ran the `run_ogusa_example.py` script.

`distributed` seemed to import fine, but I got an error related to parameter validation:
```
(ogusa-dev) jason.debacker@x86_64-apple-darwin13 run_examples % python run_ogusa_example.py
Number of workers =  4
making dir:  /Users/jason.debacker/repos/dynamic/run_examples/OUTPUT_BASELINE/SS
making dir:  /Users/jason.debacker/repos/dynamic/run_examples/OUTPUT_BASELINE/TPI
In runner, baseline is  True
Traceback (most recent call last):
  File "run_ogusa_example.py", line 126, in <module>
    main()
  File "run_ogusa_example.py", line 73, in main
    runner(**kwargs)
  File "/Users/jason.debacker/repos/dynamic/ogusa/execute.py", line 67, in runner
    client=client, num_workers=num_workers)
  File "/Users/jason.debacker/repos/dynamic/ogusa/parameters.py", line 27, in __init__
    super().__init__()
  File "/Users/jason.debacker/anaconda3/envs/ogusa-dev/lib/python3.7/site-packages/paramtools/parameters.py", line 56, in __init__
    ) = schemafactory.schemas()
  File "/Users/jason.debacker/anaconda3/envs/ogusa-dev/lib/python3.7/site-packages/paramtools/schema_factory.py", line 94, in schemas
    defaults_schema.load(self.defaults),
  File "/Users/jason.debacker/anaconda3/envs/ogusa-dev/lib/python3.7/site-packages/marshmallow/schema.py", line 723, in load
    data, many=many, partial=partial, unknown=unknown, postprocess=True
  File "/Users/jason.debacker/anaconda3/envs/ogusa-dev/lib/python3.7/site-packages/marshmallow/schema.py", line 904, in _do_load
    raise exc
marshmallow.exceptions.ValidationError: {'tau_c': {'value': {0: {'value': ['Not a valid number: [0.].']}}}}
distributed.nanny - WARNING - Restarting worker
distributed.nanny - WARNING - Restarting worker
distributed.nanny - WARNING - Restarting worker
distributed.nanny - WARNING - Restarting worker
```

I am using `paramtools 0.13.1`. 

@hdoupe Is this related to recent paramtools updates?

**Comment by hdoupe** [comment]
Yep, this is. ParamTools 0.13.1 fixes a bug that allowed array values to be set on non-array parameters. If you add `"number_dims": 1` here, then that should fix the problem: 

https://github.com/PSLmodels/OG-USA/blob/480232415697f3f4acf8ac82ac8170dc344156b4/ogusa/default_parameters.json#L647-L667

**Comment by jdebacker** [comment]
@hdoupe Thanks for the quick reply.

Do you mean like this:
```
 "tau_c": { 
     "title": "Consumption tax rate", 
     "description": "Consumption tax rate", 
     "section_1": "Fiscal Policy Parameters", 
     "section_2": "Taxes", 
     "notes": "This policy parameter represents the effective consumption tax rate from sales taxes, VATs, and excise taxes.  To capture exemptions of certain goods, this is assumed to vary by age and ability type.  It is thus a SxJ matrix.", 
     "type": "float", 
    "number_dims": 1
     "value": [ 
         { 
             "value": [ 
                 0.0 
             ] 
         } 
     ], 
     "validators": { 
         "range": { 
             "min": 0.0, 
             "max": 0.9 
         } 
     } 
 },
```

Does this need to be done for all parameters that are lists?

**Comment by hdoupe** [comment]
Yep, that's right. It looks like most of the other ones have `number_dims` set. Sorry about the API change!

**Comment by rickecon** [comment]
This issue is solved in PR #547. Thanks @jdebacker and @hdoupe.
